sensor_msgs/PointCloud2 points
jsk_recognition_msgs/BoundingBox box
---
jsk_interactive_marker/MarkerDimensions dim
geometry_msgs/PoseStamped pose_stamped